src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const basicDiscount = 50;
+  const minimalDiscount = 20;
+
+  if (days >= 7) {
+    return (days * dayPrice) - basicDiscount;
+  }
+
+  if (days >= 3) {
+    return (days * dayPrice) - minimalDiscount;
+  }
+
+  if (days < 3) {
```

Comment:
Unnecessary if statement. Since all other conditions are checked, you can use 'else' or simply return the result without any condition.